### PR TITLE
FAQ ready! [DO NOT MERGE]

### LINKED ISSUE #32: FAQ
Making the list of frequently asked questions right now. The file is on https://github.com/ScratchAddons/website/tree/faq/site/faq.md, if you wanted to know the current revision, and/or maybe wanted to help.

### PR DESCRIPTION
**Pull request by Hans5958**
Okay, here it is, the Frequently Asked Questions page is ready, at least for me.

You can check the current rev on [here](https://github.com/ScratchAddons/website/blob/faq/site/faq.md). Please leave a review about it, and suggest final changes if there is.

**Do not merge this in an instant!** I repeat, **do not merge!** Leave comments first! Let me merge it by myself.

Resolves #32.

### CODE REVIEW

File: site/faq.md [hunk 1]
```diff
@@ -1,7 +1,119 @@
...
+
+If you think a feature is missing, or you wanted to suggest an addon to the extension, send a feedback on [our feedback page](https://scratchaddons.com/feedback), or create an issue directly on [our repository](https://github.com/ScratchAddons/ScratchAddons/issues).
+
+### Where can I discuss about Scratch Addons?
+
+You can do so by doing it on [our Discussion tab](https://github.com/ScratchAddons/ScratchAddons/discussions). There, you can discuss about it and ask questions if you have difficulties.
+
+### I think Scratch Addons slows down Scratch. What can I do?
+
+Most times you turned on addons that utilise the so-called "traps" on the process. These addons often include a warning, so turn those off and try again. The warnings are look like this.
```
Comment (WorldLanguages) [inline site/faq.md:109]: Outdated. Right now I'd recommend users who feel slowdowns to open a discussion or send feedback on where exactly they feel the slowdown. This has happened in the past, for example, someone reported laggy block moving in the editor, and we ended up finding the cause and fixing it in v1.4.0. Was not related to traps at all.

File: site/faq.md [hunk 2]
```diff
@@ -1,7 +1,119 @@
...
+
+![warning](/assets/images/faq/warning.png)
+
+### Are there any consequences for enabling everything?
+
+If you have a mid-end computer, chances are the performance won't be dropped significantly, unless if you turned on addons that utilise "traps". Otherwise, it will just be a major change. Feel free to go through again and turn off addons that you don't need.
```
Comment (WorldLanguages) [inline site/faq.md:115]: Outdated.

**Review by WorldLanguages — CHANGES_REQUESTED** [review]
Looks good, except we don't use traps anymore. It's *a* trap now, just for the VM, and it's removed after the VM is trapped :P

**Comment by Hans5958** [comment]
> 
> 
> Looks good, except we don't use traps anymore. It's _a_ trap now, just for the VM, and it's removed after the VM is trapped :P

Not sure about how this new thing work, so please tell me what sentence should I put there.

**Comment by WorldLanguages** [comment]
@Hans5958 I'd just remove those 2 questions.

**Comment by Hans5958** [comment]
I'll just keep the question and decrease (or remove) the mentions of the traps. Keep in mind there are people who have low-end computers that they even get lagged when running two apps at the same time.

**Comment by Hans5958** [comment]
Alright, fixed it. Any other comments?

**Comment by WorldLanguages** [comment]
> I'll just keep the question and decrease (or remove) the mentions of the traps. Keep in mind there are people who have low-end computers that they even get lagged when running two apps at the same time.

I'm aware, but the CPU and RAM use of the extension is insignificant to those used by the browser and the Scratch player/editor.

**Comment by Hans5958** [comment]
@WorldLanguages Have you checked the latest one?

**Review by WorldLanguages — APPROVED** [review]
Changes look good

**Comment by Hans5958** [comment]
Okay, no other comments, it seems. Launch time.

**Comment by WorldLanguages** [comment]
lol the fact you merged something with [DO NOT MERGE] in its title :P

**Comment by Hans5958** [comment]
> 
> 
> lol the fact you merged something with [DO NOT MERGE] in its title :P

It's just a warning for you guys who wanted to merge by themselves

**Pull request merged**